### ISSUE TITLE
ic-prompt triggers ajax request even if canceled

### ISSUE THREAD

**Issue by dotnetchris** [LeadDyno/intercooler-js#72]
Given:

```html
<button class="btn" ic-put-to="/action" ic-prompt="Enter Customer Id">Explain</button>
```

User clicks button Explain. Alert dialog of "Enter Customer Id" is displayed. User presses cancel.

Expected: noop

Actual: request is sent to server with null ic-prompt-value header.

**Comment by carsongross** [comment]
Easy fix, will try to get to it today.  Thanks for logging it.

**Comment by carsongross** [comment]
Fixed by f1468f8ce719e26807b5609ce92293f1a4689753
